docs/sources/contribute-documentation/contribute-release-notes/index.md

```diff
@@ -115,6 +115,7 @@ When you’re ready to add a What’s new entry, complete the following steps:
            <p>If you want to view some best practices around what to write here, refer to <a href="#writing-guidelines-for-whats-new-content">Writing guidelines for What’s new</a>.</p>
            <p>Add any images and a link to your public YouTube video here.</p>
            <p>If you need more information on adding an image, refer to <a href="https://grafana.com/docs/writers-toolkit/write/image-guidelines/">Image, diagram, and screenshot guidelines</a>.</p>
+           <p>If you need to mention a feature flag, use this format: To try out Trace to profiles, enable the <code>traceToProfile</code> <a href="https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/">feature toggle</a>.</p>
```

Comment:
Would it be helpful to also include how to achieve the monospace formatting?  I don't actually know what the best practices are for formatting content written in the CMS.

Reply:
The CMS allows you to use Markdown, so folks shouldn't need an explanation. They've already been doing this anyway.